plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/TestingPinotCluster.java

```diff
@@ -65,8 +65,8 @@
 public class TestingPinotCluster
         implements Closeable
 {
-    public static final String PINOT_LATEST_IMAGE_NAME = "apachepinot/pinot:0.12.1";
-    public static final String PINOT_PREVIOUS_IMAGE_NAME = "apachepinot/pinot:0.11.0";
+    public static final String PINOT_LATEST_IMAGE_NAME = "apachepinot/pinot:1.0.0";
+    public static final String PINOT_PREVIOUS_IMAGE_NAME = "apachepinot/pinot:0.12.1";
```

Comment:
Could you update the requirement in pinot.md?

Reply:
Update: in a follow up pr, we can remove support for previous version. We spoke with Pinot committers and will rely on Pinot for backwards compatibility, going forward Pinot 1.x.x+ will manage compatibility. This also simplifies the code and removes some dependency issues.